packages/vkui/jest.config.a11y.js

```diff
@@ -0,0 +1,16 @@
+const config = require('./jest.config.js');
+
+module.exports = {
+  ...config,
+  displayName: 'a11y',
+  testMatch: ['**/*.test.a11y.{ts,tsx}'],
```

Comment:
Давай сделаем вот так:

```suggestion
  testMatch: ['**/*.a11y.test.{ts,tsx}'],
```

- jest плагин из коробки не понимает что это тесты
- vscode из коробки не понимает что это тесты
- надо проверить что сами тестовые файлы не попадают в покрытие

Reply:
> надо проверить что сами тестовые файлы не попадают в покрытие

Я проверила, не попадают!

> jest плагин из коробки не понимает что это тесты
> vscode из коробки не понимает что это тесты 

О, хороший аргумент.

С e2e такая же штука, кстати, может, сделать задачку под их переименование?

Reply:
c e2e тестами сложнее, ибо локально они плохо работают :)

Reply:
Штош, тогда пока так оставим. :(

✔️ (переименовала и в конфиге, и залетевшие тесты)